src/CorsProfile/CorsProfile.php

```diff
@@ -19,4 +19,8 @@ public function addPreflightHeaders($response);
     public function maxAge(): int;
 
     public function isAllowed(): bool;
+
+    public function forbiddenMessage(): string;
```

Comment:
Unfortunately adding methods to an interface is a breaking change.

Let's avoid breaking changes by just using the new config values directly in `forbiddenResponse`.

Reply:
@freekmurze Thanks for the review. I've removed the two methods in question and updated `forbiddenResponse` as requested.